docs/OCP_CI_Tutorials/Scenarios/Provision_Guide.md

```diff
@@ -0,0 +1,70 @@
+# (WIP) OpenShift CI Scenario Provision Guide<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Overview](#overview)
+- [Create Foundational OpenShift CI Scenario Files](#create-foundational-openshift-ci-scenario-files)
+  - [1. Create a directory within the config directory](#1-create-a-directory-within-the-config-directory)
+  - [2. Create directories for the scenario in the step-registry](#2-create-directories-for-the-scenario-in-the-step-registry)
+  - [3. Create the Scenario Chain](#3-create-the-scenario-chain)
+  - [4. Create Config in Directory from Step #1.](#4-create-config-in-directory-from-step-1)
+    - [OCP Image Version](#ocp-image-version)
+    - [Test stanza creation](#test-stanza-creation)
+  - [5. Submit PR](#5-submit-pr)
+
+## Overview
+Here we will be submitting our first PR to the [openshift/release](https://github.com/openshift/release) repo.
```

Comment:
Are we still planning on splitting these into separate PRs?

Reply:
No, we want a single PR, good catch.

I had left the submit PR process here because it'll be where someone needs to learn how to submit and work within their first PR but I will update to be clear that we want 1 PR per scenario.